# Polish translation for duplicity
# Copyright (c) 2010 Rosetta Contributors and Canonical Ltd 2010
# This file is distributed under the same license as the duplicity package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2010.
#
msgid ""
msgstr ""
"Project-Id-Version: duplicity\n"
"Report-Msgid-Bugs-To: Kenneth Loafman <kenneth@loafman.com>\n"
"POT-Creation-Date: 2020-03-07 09:49-0600\n"
"PO-Revision-Date: 2018-02-23 13:10+0000\n"
"Last-Translator: Łukasz Cieliński <lukasc@mp.pl>\n"
"Language-Team: Polish <pl@li.org>\n"
"Language: pl\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=n==1 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 "
"|| n%100>=20) ? 1 : 2;\n"
"X-Launchpad-Export-Date: 2020-03-18 06:31+0000\n"
"X-Generator: Launchpad (build 3a6db24bbe7280ec09bae73384238390fcc98ad3)\n"

#: ../bin/duplicity:120
msgid "INT intercepted...exiting."
msgstr "Przechwycono sygnał INT ... zakańczanie."

#: ../bin/duplicity:128
#, python-format
msgid "GPG error detail: %s"
msgstr "Szczegóły błędu GPG: %s"

#: ../bin/duplicity:138
#, python-format
msgid "User error detail: %s"
msgstr "Szczegóły błędu użytkownika: %s"

#: ../bin/duplicity:148
#, python-format
msgid "Backend error detail: %s"
msgstr "Szczegóły błędu backendu: %s"

#: ../bin/rdiffdir:64 ../duplicity/commandline.py:236
#, python-format
msgid "Error opening file %s"
msgstr "Błąd podczas próby otwarcia pliku %s"

#: ../bin/rdiffdir:131
#, python-format
msgid "File %s already exists, will not overwrite."
msgstr "Plik %s już istnieje i nie zostanie nadpisany."

#: ../duplicity/asyncscheduler.py:70
#, python-format
msgid "instantiating at concurrency %d"
msgstr "wykonywanie współbieżne (%d zadań)"

#: ../duplicity/asyncscheduler.py:97
msgid "inserting barrier"
msgstr "oczekiwanie wątków na barierze"

#: ../duplicity/asyncscheduler.py:146
msgid "running task synchronously (asynchronicity disabled)"
msgstr "wykonywania zadań sekwencyjnie (współbieżność wyłączona)"

#: ../duplicity/asyncscheduler.py:152
msgid "scheduling task for asynchronous execution"
msgstr "zgłaszanie zadania do współbieżnego wykonania"

#: ../duplicity/asyncscheduler.py:181
msgid "task completed successfully"
msgstr "zadanie zakończone pomyślnie"

#: ../duplicity/asyncscheduler.py:192
msgid ""
"a previously scheduled task has failed; propagating the result immediately"
msgstr ""
"wcześniej zgłoszone zadanie zakończone błędem; natychmiastowa propagacja "
"błędu"

#: ../duplicity/asyncscheduler.py:215 ../duplicity/asyncscheduler.py:236
#, python-format
msgid "active workers = %d"
msgstr "liczba aktywnych zadań = %d"

#: ../duplicity/asyncscheduler.py:256
#, python-format
msgid "task execution done (success: %s)"
msgstr "zadanie zakończone (sukces: %s)"

#: ../duplicity/backend.py:104
#, python-format
msgid "Import of %s %s"
msgstr "Importowanie %s: %s"

#: ../duplicity/backend.py:213
#, python-format
msgid "Could not initialize backend: %s"
msgstr "Inicjalizacja backendu nieudana: %s"

#: ../duplicity/backend.py:392
#, python-format
msgid "Backtrace of previous error: %s"
msgstr "Szczegóły poprzedniego błędu: %s"

#: ../duplicity/backend.py:408
#, python-format
msgid "Giving up after %s attempts. %s: %s"
msgstr "Rezygnacja po %s próbach. %s:%s"

#: ../duplicity/backend.py:412
#, python-format
msgid "Attempt %s failed. %s: %s"
msgstr "Próba %s nieudana. %s:%s"

#: ../duplicity/backend.py:506
#, python-format
msgid "Reading results of '%s'"
msgstr "Wczytywanie wyników wykonania '%s'"

#: ../duplicity/backend.py:532
#, python-format
msgid "Writing %s"
msgstr "Zapisywanie %s"

#: ../duplicity/backend.py:573
#, python-format
msgid "File %s not found locally after get from backend"
msgstr "Plik %s nieodnaleziony lokalnie po pobraniu z backendu"

#: ../duplicity/backends/giobackend.py:109
#, python-format
msgid "Connection failed, please check your password: %s"
msgstr "Połączenie nie powiodło się; sprawdź podane hasło: %s"

#: ../duplicity/backends/multibackend.py:97
#, python-format
msgid "MultiBackend: Could not parse query string %s: %s "
msgstr ""

#: ../duplicity/backends/multibackend.py:106
#, python-format
msgid "MultiBackend: Invalid query string %s: more than one value for %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:111
#, python-format
msgid "MultiBackend: Invalid query string %s: unknown parameter %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:161
#: ../duplicity/backends/multibackend.py:166
#, python-format
msgid "MultiBackend: illegal value for %s: %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:177
#, python-format
msgid "MultiBackend: Url %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:181
#, python-format
msgid "MultiBackend: Could not load config file %s: %s "
msgstr ""

#: ../duplicity/backends/multibackend.py:190
#, python-format
msgid "MultiBackend: use store %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:195
#, python-format
msgid "MultiBackend: set env %s = %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:208
#, python-format
msgid "Multibackend: register affinity for prefix %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:249
#, python-format
msgid "MultiBackend: _put: write to store #%s (%s)"
msgstr ""

#: ../duplicity/backends/multibackend.py:262
#, python-format
msgid "MultiBackend: failed to write to store #%s (%s), try #%s, Exception: %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:269
#, python-format
msgid "MultiBackend: failed to write %s. Aborting process."
msgstr ""

#: ../duplicity/backends/multibackend.py:276
#, python-format
msgid ""
"MultiBackend: failed to write %s. Tried all backing stores and none succeeded"
msgstr ""

#: ../duplicity/backends/multibackend.py:295
#, python-format
msgid "MultiBackend: failed to get %s to %s from %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:298
#, python-format
msgid ""
"MultiBackend: failed to get %s. Tried all backing stores and none succeeded"
msgstr ""

#: ../duplicity/backends/multibackend.py:308
#, python-format
msgid "MultiBackend: %s: %d files"
msgstr ""

#: ../duplicity/backends/multibackend.py:311
#, python-format
msgid "Exception during list of %s: %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:318
#, python-format
msgid "MultiBackend: combined list: %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:346
#, python-format
msgid "MultiBackend: failed to delete %s from %s"
msgstr ""

#: ../duplicity/backends/multibackend.py:350
#, python-format
msgid ""
"MultiBackend: failed to delete %s. Tried all backing stores and none "
"succeeded"
msgstr ""

#: ../duplicity/backends/pydrivebackend.py:150
#, python-format
msgid "PyDrive backend: multiple files called '%s'."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:66
msgid "Missing socket or ssl python modules."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:84
#, python-format
msgid "Cacert database file '%s' is not readable."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:105
msgid ""
"Option '--ssl-cacert-path' is not supported with python 2.7.8 and below."
msgstr ""
"Opcja '--ssl-cacert-path' jest nieobsługiwana z python 2.7.8 i poniższymi."

#: ../duplicity/backends/webdavbackend.py:109
#, python-format
msgid ""
"For certificate verification with python 2.7.8 or earlier a cacert database\n"
"file is needed in one of these locations: %s\n"
"Hints:\n"
"Consult the man page, chapter 'SSL Certificate Verification'.\n"
"Consider using the options --ssl-cacert-file, --ssl-no-check-certificate ."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:155
#, python-format
msgid "Using WebDAV protocol %s"
msgstr "Używanie protokołu WebDAV %s"

#: ../duplicity/backends/webdavbackend.py:156
#, python-format
msgid "Using WebDAV host %s port %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:158
#, python-format
msgid "Using WebDAV directory %s"
msgstr "Używanie katalogu WebDAV %s"

#: ../duplicity/backends/webdavbackend.py:189
#, python-format
msgid "WebDAV create connection on '%s'"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:200
#, python-format
msgid "WebDAV Unknown URI scheme: %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:219
#, python-format
msgid "WebDAV %s %s request with headers: %s "
msgstr ""

#: ../duplicity/backends/webdavbackend.py:220
#: ../duplicity/backends/webdavbackend.py:243
#, python-format
msgid "WebDAV data length: %s "
msgstr ""

#: ../duplicity/backends/webdavbackend.py:223
#, python-format
msgid "WebDAV response status %s with reason '%s'."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:229
#, python-format
msgid "WebDAV redirect to: %s "
msgstr ""

#: ../duplicity/backends/webdavbackend.py:231
msgid "WebDAV redirected 10 times. Giving up."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:236
msgid "WebDAV missing location header in redirect response."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:241
msgid "WebDAV retry request with authentification headers."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:242
#, python-format
msgid "WebDAV %s %s request2 with headers: %s "
msgstr ""

#: ../duplicity/backends/webdavbackend.py:246
#, python-format
msgid "WebDAV response2 status %s with reason '%s'."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:263
msgid ""
"python-kerberos needed to use kerberos                           "
"authorization, falling back to basic auth."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:267
#, python-format
msgid ""
"Kerberos authorization failed: %s.                          Falling back to "
"basic auth."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:365
#, python-format
msgid "Creating missing directory %s"
msgstr "Tworzenie brakującego katalogu %s"

#: ../duplicity/backends/webdavbackend.py:369
#, python-format
msgid "WebDAV MKCOL %s failed: %s %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:382
#, python-format
msgid "WebDAV path decoding and translation: %s -> %s"
msgstr ""

#: ../duplicity/backends/webdavbackend.py:427
#, python-format
msgid "WebDAV GET Bad status code %s reason %s."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:449
#, python-format
msgid "WebDAV PUT Bad status code %s reason %s."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:469
#, python-format
msgid "WebDAV DEL Bad status code %s reason %s."
msgstr ""

#: ../duplicity/commandline.py:79
#, python-format
msgid ""
"Warning: Option %s is pending deprecation and will be removed in a future "
"release.\n"
"Use of default filenames is strongly suggested."
msgstr ""
"Ostrzeżenie: Opcja %s jest przestarzała i zostanie wkrótce usunięta.\n"
"Sugeruje się używanie standardowych nazw plików."

#: ../duplicity/commandline.py:86
#, python-format
msgid ""
"Warning: Option %s is pending deprecation and will be removed in a future "
"release.\n"
"--include-filelist and --exclude-filelist now accept globbing characters and "
"should be used instead."
msgstr ""

#: ../duplicity/commandline.py:96
#, python-format
msgid ""
"Warning: Option %s is pending deprecation and will be removed in a future "
"release.\n"
"On many GNU/Linux systems, stdin is represented by /dev/stdin and\n"
"--include-filelist=/dev/stdin or --exclude-filelist=/dev/stdin could\n"
"be used as a substitute."
msgstr ""

#. Used in usage help to represent a Unix-style path name. Example:
#. --archive-dir <path>
#: ../duplicity/commandline.py:257 ../duplicity/commandline.py:267
#: ../duplicity/commandline.py:288 ../duplicity/commandline.py:359
#: ../duplicity/commandline.py:377 ../duplicity/commandline.py:609
#: ../duplicity/commandline.py:642 ../duplicity/commandline.py:855
msgid "path"
msgstr "ścieżka"

#. Used in usage help to represent an ID for a GnuPG key. Example:
#. --encrypt-key <gpg_key_id>
#. Used in usage help to represent an ID for a hidden GnuPG key. Example:
#. --hidden-encrypt-key <gpg_key_id>
#. Used in usage help to represent an ID for a GnuPG key. Example:
#. --encrypt-key <gpg_key_id>
#: ../duplicity/commandline.py:283 ../duplicity/commandline.py:290
#: ../duplicity/commandline.py:383 ../duplicity/commandline.py:593
#: ../duplicity/commandline.py:828
msgid "gpg-key-id"
msgstr "id_klucza_gpg"

#. Used in usage help to represent a "glob" style pattern for
#. matching one or more files, as described in the documentation.
#. Example:
#. --exclude <shell_pattern>
#: ../duplicity/commandline.py:298 ../duplicity/commandline.py:408
#: ../duplicity/commandline.py:878
msgid "shell_pattern"
msgstr "wzorzec_powłoki"

#. Used in usage help to represent the name of a file. Example:
#. --log-file <filename>
#: ../duplicity/commandline.py:304 ../duplicity/commandline.py:313
#: ../duplicity/commandline.py:320 ../duplicity/commandline.py:410
#: ../duplicity/commandline.py:417 ../duplicity/commandline.py:430
#: ../duplicity/commandline.py:824
msgid "filename"
msgstr "nazwa_pliku"

#. Used in usage help to represent a regular expression (regexp).
#: ../duplicity/commandline.py:327 ../duplicity/commandline.py:421
msgid "regular_expression"
msgstr "wyrażenie_regularne"

#. Used in usage help to represent a time spec for a previous
#. point in time, as described in the documentation. Example:
#. duplicity remove-older-than time [options] target_url
#: ../duplicity/commandline.py:331 ../duplicity/commandline.py:371
#: ../duplicity/commandline.py:497 ../duplicity/commandline.py:910
msgid "time"
msgstr "data"

#. Used in usage help. (Should be consistent with the "Options:"
#. header.) Example:
#. duplicity [full|incremental] [options] source_dir target_url
#: ../duplicity/commandline.py:379 ../duplicity/commandline.py:477
#: ../duplicity/commandline.py:500 ../duplicity/commandline.py:601
#: ../duplicity/commandline.py:843
msgid "options"
msgstr "opcje"

#: ../duplicity/commandline.py:394
#, python-format
msgid ""
"Running in 'ignore errors' mode due to %s; please re-consider if this was "
"not intended"
msgstr ""
"Działanie w trybie 'ignorowania błędów' z powodu %s. Upewnij się, że tego "
"właśnie chciałeś."

#. Used in usage help to represent an imap mailbox
#: ../duplicity/commandline.py:406
msgid "imap_mailbox"
msgstr "skrzynka_imap"

#: ../duplicity/commandline.py:424
msgid "file_descriptor"
msgstr "deskryptor_pliku"

#. Used in usage help to represent a desired number of
#. something. Example:
#. --num-retries <number>
#: ../duplicity/commandline.py:435 ../duplicity/commandline.py:457
#: ../duplicity/commandline.py:474 ../duplicity/commandline.py:483
#: ../duplicity/commandline.py:533 ../duplicity/commandline.py:538
#: ../duplicity/commandline.py:542 ../duplicity/commandline.py:561
#: ../duplicity/commandline.py:567 ../duplicity/commandline.py:571
#: ../duplicity/commandline.py:637 ../duplicity/commandline.py:838
msgid "number"
msgstr "liczba"

#. Used in usage help (noun)
#: ../duplicity/commandline.py:438
msgid "backup name"
msgstr "nazwa kopii zapasowej"

#: ../duplicity/commandline.py:556
msgid "policy"
msgstr ""

#: ../duplicity/commandline.py:574
msgid "Hot|Cool|Archive"
msgstr ""

#. noun
#: ../duplicity/commandline.py:577 ../duplicity/commandline.py:580
#: ../duplicity/commandline.py:809
msgid "command"
msgstr "polecenie"

#: ../duplicity/commandline.py:583
msgid "pyrax|cloudfiles"
msgstr "pyrax|cloudfiles"

#: ../duplicity/commandline.py:604
msgid "pem formatted bundle of certificate authorities"
msgstr "plik PEM z certyfikatami"

#: ../duplicity/commandline.py:605
msgid "path to a folder with certificate authority files"
msgstr ""

#. Used in usage help. Example:
#. --timeout <seconds>
#. Used in usage help. Example:
#. --backend-retry-delay <seconds>
#. Used in usage help. Example:
#. --timeout <seconds>
#: ../duplicity/commandline.py:614 ../duplicity/commandline.py:648
#: ../duplicity/commandline.py:872
msgid "seconds"
msgstr "sekund(-y)"

#. abbreviation for "character" (noun)
#: ../duplicity/commandline.py:620 ../duplicity/commandline.py:806
msgid "char"
msgstr "znak"

#: ../duplicity/commandline.py:772
#, python-format
msgid "Using archive dir: %s"
msgstr "Używanie katalogu archiwum: %s"

#: ../duplicity/commandline.py:773
#, python-format
msgid "Using backup name: %s"
msgstr "Używanie nazwy kopii zapasowej: %s"

#: ../duplicity/commandline.py:780
#, python-format
msgid "Command line error: %s"
msgstr "Błąd linii poleceń: %s"

#: ../duplicity/commandline.py:781
msgid "Enter 'duplicity --help' for help screen."
msgstr "Wykonaj 'duplicity --help', by uzyskać pomoc."

#. Used in usage help to represent a Unix-style path name. Example:
#. rsync://user[:password]@other_host[:port]//absolute_path
#: ../duplicity/commandline.py:794
msgid "absolute_path"
msgstr "ścieżka_absolutna"

#. Used in usage help. Example:
#. tahoe://alias/some_dir
#: ../duplicity/commandline.py:798
msgid "alias"
msgstr "alias"

#. Used in help to represent a "bucket name" for Amazon Web
#. Services' Simple Storage Service (S3). Example:
#. s3://other.host/bucket_name[/prefix]
#: ../duplicity/commandline.py:803
msgid "bucket_name"
msgstr "nazwa_bucketa"

#. Used in usage help to represent the name of a container in
#. Amazon Web Services' Cloudfront. Example:
#. cf+http://container_name
#: ../duplicity/commandline.py:814
msgid "container_name"
msgstr "nazwa_kontenera"

#. noun
#: ../duplicity/commandline.py:817
msgid "count"
msgstr "liczba"

#. Used in usage help to represent the name of a file directory
#: ../duplicity/commandline.py:820
msgid "directory"
msgstr "katalog"

#. Used in usage help, e.g. to represent the name of a code
#. module. Example:
#. rsync://user[:password]@other.host[:port]::/module/some_dir
#: ../duplicity/commandline.py:833
msgid "module"
msgstr "moduł"

#. Used in usage help to represent an internet hostname. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/commandline.py:847
msgid "other.host"
msgstr "zdalny_adres"

#. Used in usage help. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/commandline.py:851
msgid "password"
msgstr "hasło"

#. Used in usage help to represent a TCP port number. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/commandline.py:859
msgid "port"
msgstr "port"

#. Used in usage help. This represents a string to be used as a
#. prefix to names for backup files created by Duplicity. Example:
#. s3://other.host/bucket_name[/prefix]
#: ../duplicity/commandline.py:864
msgid "prefix"
msgstr "prefiks"

#. Used in usage help to represent a Unix-style path name. Example:
#. rsync://user[:password]@other.host[:port]/relative_path
#: ../duplicity/commandline.py:868
msgid "relative_path"
msgstr "ścieżka_względna"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory. Example:
#. file:///some_dir
#: ../duplicity/commandline.py:883
msgid "some_dir"
msgstr "katalog"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory where files will be
#. coming FROM. Example:
#. duplicity [full|incremental] [options] source_dir target_url
#: ../duplicity/commandline.py:889
msgid "source_dir"
msgstr "katalog_źródłowy"

#. Used in usage help to represent a URL files will be coming
#. FROM. Example:
#. duplicity [restore] [options] source_url target_dir
#: ../duplicity/commandline.py:894
msgid "source_url"
msgstr "url_źródłowy"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory. where files will be
#. going TO. Example:
#. duplicity [restore] [options] source_url target_dir
#: ../duplicity/commandline.py:900
msgid "target_dir"
msgstr "katalog_docelowy"

#. Used in usage help to represent a URL files will be going TO.
#. Example:
#. duplicity [full|incremental] [options] source_dir target_url
#: ../duplicity/commandline.py:905
msgid "target_url"
msgstr "url_docelowy"

#. Used in usage help to represent a user name (i.e. login).
#. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/commandline.py:915
msgid "user"
msgstr "użytkownik"

#. account id for b2. Example: b2://account_id@bucket/
#: ../duplicity/commandline.py:918
msgid "account_id"
msgstr ""

#. application_key for b2.
#. Example: b2://account_id:application_key@bucket/
#: ../duplicity/commandline.py:922
msgid "application_key"
msgstr ""

#. remote name for rclone.
#. Example: rclone://remote:/some_dir
#: ../duplicity/commandline.py:926 ../duplicity/dup_collections.py:511
msgid "remote"
msgstr "zdalna"

#. Header in usage help
#: ../duplicity/commandline.py:945
msgid "Backends and their URL formats:"
msgstr "Backendy oraz ich formaty URL:"

#. Header in usage help
#: ../duplicity/commandline.py:978
msgid "Commands:"
msgstr "Polecenia:"

#: ../duplicity/commandline.py:1003
#, python-format
msgid "Specified archive directory '%s' does not exist, or is not a directory"
msgstr "Podana nazwa katalogu '%s' nie istnieje lub nie jest katalogiem"

#: ../duplicity/commandline.py:1012
#, python-format
msgid ""
"Sign key should be an 8, 16 alt. 40 character hex string, like 'AA0E73D2'.\n"
"Received '%s' instead."
msgstr ""

#: ../duplicity/commandline.py:1072
#, python-format
msgid ""
"Restore destination directory %s already exists.\n"
"Will not overwrite."
msgstr ""
"Docelowy katalog %s już istnieje\n"
"i nie zostanie nadpisany."

#: ../duplicity/commandline.py:1077
#, python-format
msgid "Verify directory %s does not exist"
msgstr "Katalog weryfikacji %s nie istnieje"

#: ../duplicity/commandline.py:1083
#, python-format
msgid "Backup source directory %s does not exist."
msgstr "Katalog źródłowy kopii zapasowej %s nie istnieje."

#: ../duplicity/commandline.py:1114
#, python-format
msgid "Command line warning: %s"
msgstr "Ostrzeżenie dotyczące linii poleceń: %s"

#: ../duplicity/commandline.py:1114
msgid ""
"Selection options --exclude/--include\n"
"currently work only when backing up,not restoring."
msgstr ""
"Opcje --exclude oraz --include dotyczą\n"
"tylko tworzenia kopii zapasowej, a nie odzyskiwania."

#: ../duplicity/commandline.py:1150
#, python-format
msgid "GPG binary is %s, version %s"
msgstr ""

#: ../duplicity/commandline.py:1177
#, python-format
msgid ""
"Bad URL '%s'.\n"
"Examples of URL strings are \"scp://user@host.net:1234/path\" and\n"
"\"file:///usr/local\".  See the man page for more information."
msgstr ""
"Błędny URL '%s'.\n"
"Przykładowe adresy URL to np. \"scp://użytkownik@adres.net:1234/ścieżka\" "
"oraz\n"
"\"file:///usr/local\". Zobacz podręcznik systemowy, aby uzyskać więcej "
"informacji."

#: ../duplicity/commandline.py:1207
msgid "Main action: "
msgstr "Główna akcja: "

#: ../duplicity/diffdir.py:115 ../duplicity/diffdir.py:418
#, python-format
msgid "Error %s getting delta for %s"
msgstr "Błąd %s podczas pobierania zmian do %s"

#: ../duplicity/diffdir.py:130
#, python-format
msgid "Getting delta of %s and %s"
msgstr "Pobieranie zmian plików %s oraz %s"

#: ../duplicity/diffdir.py:181
#, python-format
msgid "A %s"
msgstr "A %s"

#: ../duplicity/diffdir.py:188
#, python-format
msgid "M %s"
msgstr "M %s"

#: ../duplicity/diffdir.py:210
#, python-format
msgid "Comparing %s and %s"
msgstr "Porównywanie %s oraz %s"

#: ../duplicity/diffdir.py:218
#, python-format
msgid "D %s"
msgstr "D %s"

#: ../duplicity/dup_collections.py:180 ../duplicity/dup_collections.py:194
#, python-format
msgid "BackupSet.delete: missing %s"
msgstr "BackupSet.delete: brakujący plik %s"

#: ../duplicity/dup_collections.py:219
msgid "Fatal Error: No manifests found for most recent backup"
msgstr "Krytyczny błąd: brak plików manifestu najnowszej kopii zapasowej"

#: ../duplicity/dup_collections.py:228
msgid ""
"Fatal Error: Remote manifest does not match local one.  Either the remote "
"backup set or the local archive directory has been corrupted."
msgstr ""
"Krytyczny błąd: Zdalny manifest jest różny od lokalnego. Zdalna kopia "
"zapasowa lub lokalne archiwum musiały ulec oszkodzeniu."

#: ../duplicity/dup_collections.py:236
msgid "Fatal Error: Neither remote nor local manifest is readable."
msgstr ""
"Krytyczny błąd: zarówno zdalny jak i lokalny manifest jest nieczytelny."

#: ../duplicity/dup_collections.py:247
#, python-format
msgid "Processing local manifest %s (%s)"
msgstr ""

#: ../duplicity/dup_collections.py:259
#, python-format
msgid "Error processing remote manifest (%s): %s"
msgstr ""

#: ../duplicity/dup_collections.py:262
#, python-format
msgid "Processing remote manifest %s (%s)"
msgstr ""

#: ../duplicity/dup_collections.py:358
msgid "Preferring Backupset over previous one!"
msgstr "Wybór tej kopii zapasowej zamiast poprzedniej"

#: ../duplicity/dup_collections.py:361
#, python-format
msgid "Ignoring incremental Backupset (start_time: %s; needed: %s)"
msgstr ""
"Pominięcie przyrostowej kopii zapasowej (data rozpoczęcia: %s; oczekiwano: "
"%s)"

#: ../duplicity/dup_collections.py:366
#, python-format
msgid "Added incremental Backupset (start_time: %s / end_time: %s)"
msgstr ""
"Dodano przyrostową kopię zapasową (data rozpoczęcia: %s / data ukończenia: "
"%s)"

#: ../duplicity/dup_collections.py:436
msgid "Chain start time: "
msgstr "Data utworzenia łańcucha: "

#: ../duplicity/dup_collections.py:437
msgid "Chain end time: "
msgstr "Data ukończenia łańcucha: "

#: ../duplicity/dup_collections.py:438
#, python-format
msgid "Number of contained backup sets: %d"
msgstr "Liczba zawartych kopii zapasowych: %d"

#: ../duplicity/dup_collections.py:440
#, python-format
msgid "Total number of contained volumes: %d"
msgstr "Całkowita liczba zawartych woluminów: %d"

#: ../duplicity/dup_collections.py:442 ../duplicity/dup_collections.py:1231
msgid "Type of backup set:"
msgstr "Typ kopii zapasowej:"

#: ../duplicity/dup_collections.py:442 ../duplicity/dup_collections.py:1231
msgid "Time:"
msgstr "Data:"

#: ../duplicity/dup_collections.py:442
msgid "Num volumes:"
msgstr "Liczba woluminów:"

#: ../duplicity/dup_collections.py:446 ../duplicity/dup_collections.py:1237
msgid "Full"
msgstr "Pełny"

#: ../duplicity/dup_collections.py:449 ../duplicity/dup_collections.py:1239
msgid "Incremental"
msgstr "Przyrostowy"

#: ../duplicity/dup_collections.py:509
msgid "local"
msgstr "lokalna"

#: ../duplicity/dup_collections.py:666
msgid "Collection Status"
msgstr "Status kolekcji"

#: ../duplicity/dup_collections.py:668
#, python-format
msgid "Connecting with backend: %s"
msgstr "Łączenie z backendem: %s"

#: ../duplicity/dup_collections.py:670
#, python-format
msgid "Archive dir: %s"
msgstr "Katalog archiwum: %s"

#: ../duplicity/dup_collections.py:673
#, python-format
msgid "Found %d secondary backup chain."
msgid_plural "Found %d secondary backup chains."
msgstr[0] "Znaleziono %d drugorzędny łańcuch kopii zapasowych."
msgstr[1] "Znaleziono %d drugorzędne łańcuchy kopii zapasowych."
msgstr[2] "Znaleziono %d drugorzędnych łańcuchów kopii zapasowych."

#: ../duplicity/dup_collections.py:678
#, python-format
msgid "Secondary chain %d of %d:"
msgstr "Drugorzędny łańcuch %d spośród %d"

#: ../duplicity/dup_collections.py:684
msgid "Found primary backup chain with matching signature chain:"
msgstr "Znaleziono podstawowy łańuch kopii zapasowej z pasującym podpisem:"

#: ../duplicity/dup_collections.py:688
msgid "No backup chains with active signatures found"
msgstr "Nie znaleziono łańcuchów kopii zapasowej z aktywnymi podpisami"

#: ../duplicity/dup_collections.py:691
#, python-format
msgid "Also found %d backup set not part of any chain,"
msgid_plural "Also found %d backup sets not part of any chain,"
msgstr[0] ""
"Dodatkowo znaleziono %d kopię zapasową nie będącą częścią żadnego łańcucha."
msgstr[1] ""
"Dodatkowo znaleziono %d kopie zapasowe nie będące częścią żadnego łańcucha."
msgstr[2] ""
"Dodatkowo znaleziono %d kopii zapasowych nie będących częścią żadnego "
"łańcucha."

#: ../duplicity/dup_collections.py:695
#, python-format
msgid "and %d incomplete backup set."
msgid_plural "and %d incomplete backup sets."
msgstr[0] "i %d niepełną kopię zapasową"
msgstr[1] "i %d niepełne kopie zapasowe"
msgstr[2] "i %d niepełnych kopii zapasowych"

#. "cleanup" is a hard-coded command, so do not translate it
#: ../duplicity/dup_collections.py:700
msgid "These may be deleted by running duplicity with the \"cleanup\" command."
msgstr "Te kopie można usunąć uruchamiając duplicity z komendą \"cleanup\"."

#: ../duplicity/dup_collections.py:703
msgid "No orphaned or incomplete backup sets found."
msgstr "Nie znaleziono osieroconych ani niepełnych kopii zapasowych."

#: ../duplicity/dup_collections.py:719
#, python-format
msgid "%d file exists on backend"
msgid_plural "%d files exist on backend"
msgstr[0] "backend zawiera %d plik"
msgstr[1] "backend zawiera %d pliki"
msgstr[2] "backend zawiera %d plików"

#: ../duplicity/dup_collections.py:729
#, python-format
msgid "%d file exists in cache"
msgid_plural "%d files exist in cache"
msgstr[0] "pamięć podręczna zawiera %d plik"
msgstr[1] "pamięć podręczna zawiera %d pliki"
msgstr[2] "pamięć podręczna zawiera %d plików"

#: ../duplicity/dup_collections.py:782
msgid "Warning, discarding last backup set, because of missing signature file."
msgstr ""
"Uwaga, usuwanie ostatniej kopii zapasowej z powodu braku pliku podpisu."

#: ../duplicity/dup_collections.py:805
msgid "Warning, found the following local orphaned signature file:"
msgid_plural "Warning, found the following local orphaned signature files:"
msgstr[0] "Uwaga, znaleziono lokalny, osierocony  plik podpisu:"
msgstr[1] "Uwaga, znaleziono lokalne, osierocone pliki podpisów:"
msgstr[2] "Uwaga, znaleziono lokalne, osierocone pliki podpisów:"

#: ../duplicity/dup_collections.py:814
msgid "Warning, found the following remote orphaned signature file:"
msgid_plural "Warning, found the following remote orphaned signature files:"
msgstr[0] "Uwaga, znaleziono zdalny, osierocony plik podpisu:"
msgstr[1] "Uwaga, znaleziono zdalne, osierocone pliki podpisów:"
msgstr[2] "Uwaga, znaleziono zdalne, osierocone pliki podpisów:"

#: ../duplicity/dup_collections.py:823
msgid "Warning, found signatures but no corresponding backup files"
msgstr "Uwaga, znaleziono pliki podpisów, ale bez plików kopii zapasowej"

#: ../duplicity/dup_collections.py:827
msgid ""
"Warning, found incomplete backup sets, probably left from aborted session"
msgstr ""
"Uwaga, znaleziono nieukończone kopie zapasowe, prawdopodobnie pozostałości "
"po przerwanej sesji"

#: ../duplicity/dup_collections.py:831
msgid "Warning, found the following orphaned backup file:"
msgid_plural "Warning, found the following orphaned backup files:"
msgstr[0] "Uwaga, znaleziono osierocony plik kopii zapasowej:"
msgstr[1] "Uwaga, znaleziono osierocone pliki kopii zapasowych:"
msgstr[2] "Uwaga, znaleziono osierocone pliki kopii zapasowych:"

#: ../duplicity/dup_collections.py:848
#, python-format
msgid "Extracting backup chains from list of files: %s"
msgstr "Uzyskiwanie łańcuchów kopii zapasowych z listy plików: %s"

#: ../duplicity/dup_collections.py:860
#, python-format
msgid "File %s is part of known set"
msgstr "Plik %s jest częścią znanej kopii zapasowej"

#: ../duplicity/dup_collections.py:863
#, python-format
msgid "File %s is not part of a known set; creating new set"
msgstr "Plik %s nie jest częścią znanej kopii zapasowej; tworzenie nowej kopii"

#: ../duplicity/dup_collections.py:868
#, python-format
msgid "Ignoring file (rejected by backup set) '%s'"
msgstr "Pominięcie pliku '%s' (wyrzucony z kopii zapasowej)"

#: ../duplicity/dup_collections.py:884
#, python-format
msgid "Found backup chain %s"
msgstr "Znaleziono łańcuch kopii zapasowych %s"

#: ../duplicity/dup_collections.py:889
#, python-format
msgid "Added set %s to pre-existing chain %s"
msgstr "Dodano kopię zapasową %s  do istniejącego łańucha %s"

#: ../duplicity/dup_collections.py:893
#, python-format
msgid "Found orphaned set %s"
msgstr "Znaleziono osierocony łańcuch %s"

#: ../duplicity/dup_collections.py:1043
#, python-format
msgid ""
"No signature chain for the requested time. Using oldest available chain, "
"starting at time %s."
msgstr ""

#: ../duplicity/dup_collections.py:1229
#, python-format
msgid "File: %s"
msgstr "Plik: %s"

#: ../duplicity/dup_collections.py:1230
#, python-format
msgid "Total number of backup: %d"
msgstr "Łączna ilość kopii zapasowych: %d"

#: ../duplicity/dup_collections.py:1231
msgid "Type of file change:"
msgstr ""

#: ../duplicity/dup_main.py:121
msgid "Reuse configured PASSPHRASE as SIGN_PASSPHRASE"
msgstr "Użyj wartość PASSPHRASE jako SIGN_PASSPHRASE"

#: ../duplicity/dup_main.py:128
msgid "Reuse configured SIGN_PASSPHRASE as PASSPHRASE"
msgstr "Użyj wartość SIGN_PASSPHRASE jako PASSPHRASE"

#: ../duplicity/dup_main.py:167
msgid "PASSPHRASE variable not set, asking user."
msgstr "Wartość PASSPHRASE nieustawiona, prośba o podanie."

#: ../duplicity/dup_main.py:182
msgid "GnuPG passphrase for signing key:"
msgstr "Hasło GnuPG do klucza podpisu cyfrowego:"

#: ../duplicity/dup_main.py:187
msgid "GnuPG passphrase:"
msgstr "Hasło GnuPG:"

#: ../duplicity/dup_main.py:192
msgid "Retype passphrase for signing key to confirm: "
msgstr "Podaj ponownie hasło do klucza podpisu cyfrowego, aby potwierdzić: "

#: ../duplicity/dup_main.py:194
msgid "Retype passphrase to confirm: "
msgstr "Podaj ponownie, aby potwierdzić: "

#: ../duplicity/dup_main.py:197
msgid "First and second passphrases do not match!  Please try again."
msgstr "Hasła nie zgadzają się! Proszę je podać ponownie."

#: ../duplicity/dup_main.py:204
msgid ""
"Cannot use empty passphrase with symmetric encryption!  Please try again."
msgstr ""
"Hasło do szyfru symetrycznego nie może być puste. Proszę je podać ponownie."

#: ../duplicity/dup_main.py:261
#, python-format
msgid ""
"File %s complete in backup set.\n"
"Continuing restart on file %s."
msgstr ""
"Plik %s zakończony w kopii zapasowej.\n"
"Kontynuacja od pliku %s."

#: ../duplicity/dup_main.py:270
#, python-format
msgid ""
"File %s missing in backup set.\n"
"Continuing restart on file %s."
msgstr ""
"Plik %s nieobecny w kopii zapasowej.\n"
"Kontynuacja od pliku %s."

#: ../duplicity/dup_main.py:319
#, python-format
msgid "File %s was corrupted during upload."
msgstr "Plik %s został uszkodzony podczas przesyłania"

#: ../duplicity/dup_main.py:359
msgid ""
"Restarting backup, but current encryption settings do not match original "
"settings"
msgstr ""
"Wznawianie tworzenia kopii zapasowej, mimo że aktualne ustawienia "
"szyfrowania są inne niż oryginalne."

#: ../duplicity/dup_main.py:382
#, python-format
msgid "Restarting after volume %s, file %s, block %s"
msgstr "Wznawianie za woluminem %s, plikiem %s, blokiem %s"

#: ../duplicity/dup_main.py:452
#, python-format
msgid "Processed volume %d"
msgstr "Przetworzono wolumin %d"

#: ../duplicity/dup_main.py:602
msgid ""
"Fatal Error: Unable to start incremental backup.  Old signatures not found "
"and incremental specified"
msgstr ""
"Błąd krytyczny: Nie można uruchomić tworzenia kopii przyrostowej. Podano "
"kopię przyrostową, a nie odnaleziono poprzednich sygnatur"

#: ../duplicity/dup_main.py:606
msgid "No signatures found, switching to full backup."
msgstr "Nie odnaleziono sygnatur, zmiana na pełną kopię zapasową."

#: ../duplicity/dup_main.py:620
msgid "Backup Statistics"
msgstr "Statystyki kopii zapasowej"

#: ../duplicity/dup_main.py:726
#, python-format
msgid "%s not found in archive - no files restored."
msgstr "%s nie odnaleziony w archiwum - nie przywrócono plików."

#: ../duplicity/dup_main.py:730
msgid "No files found in archive - nothing restored."
msgstr "Nie znaleziono żadnych plików w archiwum - nic nie odtworzono."

#: ../duplicity/dup_main.py:763
#, python-format
msgid "Processed volume %d of %d"
msgstr "Przetworzono wolumin %d z %d"

#: ../duplicity/dup_main.py:797
#, python-format
msgid "Invalid data - %s hash mismatch for file:"
msgstr "Błędne dane - %s błędna wartość funkcji skrótu dla pliku:"

#: ../duplicity/dup_main.py:800
#, python-format
msgid "Calculated hash: %s"
msgstr "Wyliczona wartość funkcji skrótu: %s"

#: ../duplicity/dup_main.py:801
#, python-format
msgid "Manifest hash: %s"
msgstr "Oczekiwana wartość funkcji skrótu: %s"

#: ../duplicity/dup_main.py:844
#, python-format
msgid "Volume was signed by key %s, not %s"
msgstr "Wolumin podpisano kluczem %s zamiast %s"

#: ../duplicity/dup_main.py:876
#, python-format
msgid "Verify complete: %s, %s."
msgstr "Weryfikacja ukończona: %s, %s"

#: ../duplicity/dup_main.py:877
#, python-format
msgid "%d file compared"
msgid_plural "%d files compared"
msgstr[0] "Porównano %d plik"
msgstr[1] "Porównano %d pliki"
msgstr[2] "Porównano %d plików"

#: ../duplicity/dup_main.py:879
#, python-format
msgid "%d difference found"
msgid_plural "%d differences found"
msgstr[0] "Wykryto %d różnicę"
msgstr[1] "Wykryto %d różnice"
msgstr[2] "Wykryto %d różnic"

#: ../duplicity/dup_main.py:898
msgid "No extraneous files found, nothing deleted in cleanup."
msgstr "Nie znaleziono nadmiarowych plików, czyszczenie niepotrzebne."

#: ../duplicity/dup_main.py:903
msgid "Deleting this file from backend:"
msgid_plural "Deleting these files from backend:"
msgstr[0] "Usuwanie tego pliku z archiwum:"
msgstr[1] "Usuwanie tych plików z archiwum:"
msgstr[2] "Usuwanie tych plików z archiwum:"

#: ../duplicity/dup_main.py:914
msgid "Found the following file to delete:"
msgid_plural "Found the following files to delete:"
msgstr[0] "Znaleziono plik do usunięcia:"
msgstr[1] "Znaleziono pliki do usunięcia:"
msgstr[2] "Znaleziono pliki do usunięcia:"

#: ../duplicity/dup_main.py:917
msgid "Run duplicity again with the --force option to actually delete."
msgstr "Uruchom duplicity ponownie z opcją --force, aby naprawdę usunąć pliki."

#: ../duplicity/dup_main.py:960
msgid "There are backup set(s) at time(s):"
msgstr "Znaleziono kopie zapasowe z dat:"

#: ../duplicity/dup_main.py:962
msgid "Which can't be deleted because newer sets depend on them."
msgstr "Nie mogą one zostać usunięte, ponieważ nowsze kopie od nich zależą."

#: ../duplicity/dup_main.py:966
msgid ""
"Current active backup chain is older than specified time.  However, it will "
"not be deleted.  To remove all your backups, manually purge the repository."
msgstr ""
"Aktualny łańcuch kopii zapasowych jest starszy niż podana data, ale nie "
"zostanie on usunięty. Aby usunąć wszystkie kopie zapasowe należy "
"własnoręcznie wyczyścić zawartość archiwum."

#: ../duplicity/dup_main.py:979
msgid "No old backup sets found, nothing deleted."
msgstr "Nie znaleziono starych kopii zapasowych i nic nie usunięto."

#: ../duplicity/dup_main.py:982
msgid "Deleting backup chain at time:"
msgid_plural "Deleting backup chains at times:"
msgstr[0] "Usuwanie łańcucha kopii zapasowych z datą:"
msgstr[1] "Usuwanie łańcuchów kopii zapasowych z dat:"
msgstr[2] "Usuwanie łańcuchów kopii zapasowych z dat:"

#: ../duplicity/dup_main.py:994
#, python-format
msgid "Deleting any incremental signature chain rooted at %s"
msgstr ""

#: ../duplicity/dup_main.py:996
#, python-format
msgid "Deleting any incremental backup chain rooted at %s"
msgstr ""

#: ../duplicity/dup_main.py:999
#, python-format
msgid "Deleting complete signature chain %s"
msgstr "Usuwanie całego łańcucha podpisów %s"

#: ../duplicity/dup_main.py:1001
#, python-format
msgid "Deleting complete backup chain %s"
msgstr "Usuwanie całego łańcucha kopii zapasowych %s"

#: ../duplicity/dup_main.py:1007
msgid "Found old backup chain at the following time:"
msgid_plural "Found old backup chains at the following times:"
msgstr[0] "Znaleziono stary łańcuch kopii zapasowych z datą:"
msgstr[1] "Znaleziono stary łańcuch kopii zapasowych z dat:"
msgstr[2] "Znaleziono stary łańcuch kopii zapasowych z dat:"

#: ../duplicity/dup_main.py:1011
msgid "Rerun command with --force option to actually delete."
msgstr "Uruchom duplicity ponownie z opcją --force, aby naprawdę usunąć pliki."

#: ../duplicity/dup_main.py:1034
msgid "No old backup sets found."
msgstr ""

#: ../duplicity/dup_main.py:1049
#, python-format
msgid "Signature %s already replicated"
msgstr ""

#: ../duplicity/dup_main.py:1056 ../duplicity/dup_main.py:1095
#, python-format
msgid "Replicating %s."
msgstr ""

#: ../duplicity/dup_main.py:1083
#, python-format
msgid "Backupset %s already replicated"
msgstr ""

#: ../duplicity/dup_main.py:1231
#, python-format
msgid "Deleting local %s (not authoritative at backend)."
msgstr "Usuwanie lokalnego pliku %s (który jest zbędny)"

#: ../duplicity/dup_main.py:1236
#, python-format
msgid "Unable to delete %s: %s"
msgstr "Nie można było usunąć %s: %s"

#: ../duplicity/dup_main.py:1270 ../duplicity/dup_temp.py:271
#, python-format
msgid "Failed to read %s: %s"
msgstr "Nie można odczytać %s: %s"

#: ../duplicity/dup_main.py:1284
#, python-format
msgid "Copying %s to local cache."
msgstr "Kopiowanie %s do pamięci podręcznej"

#: ../duplicity/dup_main.py:1332
msgid "Local and Remote metadata are synchronized, no sync needed."
msgstr ""
"Metadane lokalne i zdalne są identyczne. Synchronizacja plików nie jest "
"konieczna."

#: ../duplicity/dup_main.py:1337
msgid "Synchronizing remote metadata to local cache..."
msgstr "Synchronizacja zdalnych metadanych z lokalną pamięci podręczną"

#: ../duplicity/dup_main.py:1350
msgid "Sync would copy the following from remote to local:"
msgstr ""
"Synchronizacja skopiowałaby następujace pliki z lokalizacji zdalnej do "
"lokalnej:"

#: ../duplicity/dup_main.py:1353
msgid "Sync would remove the following spurious local files:"
msgstr "Synchronizacja usunęłaby następujące i zbędne pliki lokalne:"

#: ../duplicity/dup_main.py:1397
msgid "Unable to get free space on temp."
msgstr "Nie można było uzyskać tymczasowej przestrzeni dyskowej"

#: ../duplicity/dup_main.py:1405
#, python-format
msgid "Temp space has %d available, backup needs approx %d."
msgstr "Tymczasowa przestrzeń dyskowa ma %d jednostek, a potrzeba około %d."

#: ../duplicity/dup_main.py:1408
#, python-format
msgid "Temp has %d available, backup will use approx %d."
msgstr "Tymczasowa przestrzeń dyskowa ma %d z czego %d zostanie wykorzystane."

#: ../duplicity/dup_main.py:1416
msgid "Unable to get max open files."
msgstr "Nie można było uzyskać wartości limitu na otwarte pliki."

#: ../duplicity/dup_main.py:1420
#, python-format
msgid ""
"Max open files of %s is too low, should be >= 1024.\n"
"Use 'ulimit -n 1024' or higher to correct.\n"
msgstr ""
"Wartość limitu na otwarte pliki wynosi %s i jest za mała (powinna być >= "
"1024).\n"
"Użyj 'ulimit -n 1024' lub więcej, by kontynuować.\n"

#: ../duplicity/dup_main.py:1472
msgid ""
"RESTART: The first volume failed to upload before termination.\n"
"         Restart is impossible...starting backup from beginning."
msgstr ""
"RESTART: Pierwszy wolumin niepoprawnie przesłany przed ukończeniem.\n"
"         Wznowienie niemożliwe ... rozpoczynanie kopii zapasowej od początku."

#: ../duplicity/dup_main.py:1478
#, python-format
msgid ""
"RESTART: Volumes %d to %d failed to upload before termination.\n"
"         Restarting backup at volume %d."
msgstr ""
"RESTART: Woluminy od %d do %d niepoprawnie przesłane przed ukończeniem.\n"
"         Wznawianie od woluminu %d."

#: ../duplicity/dup_main.py:1485
#, python-format
msgid ""
"RESTART: Impossible backup state: manifest has %d vols, remote has %d vols.\n"
"         Restart is impossible ... duplicity will clean off the last "
"partial\n"
"         backup then restart the backup from the beginning."
msgstr ""
"RESTART: Niemożliwy stan kopii zapasowej: lokalny manifest ma %d woluminy, a "
"zdalny %d.\n"
"         Wznowienie jest niemożliwe ... duplicity wyczyści ostatnią, "
"częściową\n"
"         kopię zapasową i rozpocznie kopię zapasową od początku."

#: ../duplicity/dup_main.py:1506
msgid ""
"\n"
"PYTHONOPTIMIZE in the environment causes duplicity to fail to\n"
"recognize its own backups.  Please remove PYTHONOPTIMIZE from\n"
"the environment and rerun the backup.\n"
"\n"
"See https://bugs.launchpad.net/duplicity/+bug/931175\n"
msgstr ""
"\n"
"Zmienna środowiskowa PYTHONOPTIMIZE uniemożliiwia duplicity\n"
"rozpoznanie kopii zapasowych. Proszę usunąć PYTHONOPTIMIZE\n"
"spośród zmiennych środowiskowych i ponownie uruchomić duplicity.\n"
"\n"
"Zobacz również https://bugs.launchpad.net/duplicity/+bug/931175\n"

#: ../duplicity/dup_main.py:1529
#, python-format
msgid "Acquiring lockfile %s"
msgstr ""

#: ../duplicity/dup_main.py:1587
#, python-format
msgid "Last %s backup left a partial set, restarting."
msgstr "Ostatnia kopia zapasowa (typu %s) częściowo ukończona, wznawianie."

#: ../duplicity/dup_main.py:1591
#, python-format
msgid "Cleaning up previous partial %s backup set, restarting."
msgstr "Usuwanie nieukończonej kopii zapasowej (typu %s) oraz wznawianie."

#: ../duplicity/dup_main.py:1603
msgid "Last full backup date:"
msgstr "Data ostatniej pełnej kopii zapasowej:"

#: ../duplicity/dup_main.py:1605
msgid "Last full backup date: none"
msgstr "Data ostatniej pełnej kopii zapasowej: brak"

#: ../duplicity/dup_main.py:1608
msgid "Last full backup is too old, forcing full backup"
msgstr "Ostatnia pełna kopia zapasowa jest zbyt stara, tworzenie nowej"

#: ../duplicity/dup_main.py:1657
msgid ""
"When using symmetric encryption, the signing passphrase must equal the "
"encryption passphrase."
msgstr ""
"Korzystanie z symetrycznego szyfrowania wymaga identycznego klucza do "
"podpisu cyfrowego i szyfrowania."

#: ../duplicity/dup_time.py:63
#, python-format
msgid ""
"Bad interval string \"%s\"\n"
"\n"
"Intervals are specified like 2Y (2 years) or 2h30m (2.5 hours).  The\n"
"allowed special characters are s, m, h, D, W, M, and Y.  See the man\n"
"page for more information."
msgstr ""
"Błędna specyfikacja interwały czasu \"%s\"\n"
"\n"
"Interwały mają format 2Y (2 lata) or 2h30m (2 i pół godziny). Dostępne\n"
"jednostki to s, m, h, D, W, M, Y. Zobacz podręcznik systemowy, aby\n"
"uzyskać więcej informacji."

#: ../duplicity/dup_time.py:69
#, python-format
msgid ""
"Bad time string \"%s\"\n"
"\n"
"The acceptible time strings are intervals (like \"3D64s\"), w3-datetime\n"
"strings, like \"2002-04-26T04:22:01-07:00\" (strings like\n"
"\"2002-04-26T04:22:01\" are also acceptable - duplicity will use the\n"
"current time zone), or ordinary dates like 2/4/1997 or 2001-04-23\n"
"(various combinations are acceptable, but the month always precedes\n"
"the day)."
msgstr ""
"Błędna specyfikacja daty \"%s\"\n"
"\n"
"Daty mają postać interwałów czasowych (np. \"3D64s\"), dat w formacie\n"
"w3-datetime (np. \"2002-04-26T04:22:01-07:00\" oraz "
"\"2002-04-26T04:22:01\";\n"
"w drugim przypadku duplicity użyje systemowej strefy czasowej) oraz "
"zwyczajne\n"
"daty jak np. 2/4/1997 czy 2001-04-23 (różne możliwości są akceptowane, ale\n"
"miesiąc jest zawsze podany przed dniem)."

#: ../duplicity/gpginterface.py:238
msgid "Threading not available -- zombie processes may appear"
msgstr "Wielowątkowość niedostępna - mogą pojawić się procesy zombie"

#: ../duplicity/gpginterface.py:705
#, python-format
msgid "GPG process %d terminated before wait()"
msgstr "Proces GPG %d zakończył się przed wykonaniem wait()"

#: ../duplicity/lazy.py:352
#, python-format
msgid "Warning: oldindex %s >= newindex %s"
msgstr "Ostrzeżenie: oldindex %s >= newindex %s"

#: ../duplicity/lazy.py:427
#, python-format
msgid "Error '%s' processing %s"
msgstr "Błąd '%s' podczas przetwarzania %s"

#: ../duplicity/lazy.py:437
#, python-format
msgid "Skipping %s because of previous error"
msgstr "Pominięcie %s z powodu poprzedniego błędu"

#: ../duplicity/librsync.py:195
msgid ""
"basis_file must be a (true) file or an object whose file attribute is the "
"underlying true file object"
msgstr ""

#: ../duplicity/manifest.py:96
#, python-format
msgid ""
"Fatal Error: Backup source host has changed.\n"
"Current hostname: %s\n"
"Previous hostname: %s"
msgstr ""
"Krytyczny błąd: adres źródłowy uległ zmianie.\n"
"Aktualna nazwa: %s\n"
"Poprzednia nazwa: %s"

#: ../duplicity/manifest.py:103
#, python-format
msgid ""
"Fatal Error: Backup source directory has changed.\n"
"Current directory: %s\n"
"Previous directory: %s"
msgstr ""
"Krytyczny błąd: katalog źródłowy uległ zmianie.\n"
"Aktualny katalog: %s\n"
"Poprzedni katalog: %s"

#: ../duplicity/manifest.py:113
msgid ""
"Aborting because you may have accidentally tried to backup two different "
"data sets to the same remote location, or using the same archive directory.  "
"If this is not a mistake, use the --allow-source-mismatch switch to avoid "
"seeing this message"
msgstr ""
"Anulowanie, ponieważ chciałeś dokonać dwóch różnych kopii zapasowych "
"korzystając z jednej zdalnej lokacji lub korzystasz z jednego katalogu "
"archiwum. Jeśli to nie błąd, użyj --allow-source-mismatch, by uniknąć tej "
"wiadomości."

#: ../duplicity/manifest.py:215
#, python-format
msgid "Found manifest volume %s"
msgstr ""

#: ../duplicity/manifest.py:222
#, python-format
msgid "Found %s volumes in manifest"
msgstr ""

#: ../duplicity/manifest.py:239
#, python-format
msgid ""
"Manifest file '%s' is corrupt: File count says %d, File list contains %d"
msgstr ""

#: ../duplicity/manifest.py:258
msgid "Manifests not equal because different volume numbers"
msgstr "Manifesty różne, bo zawierają listy woluminów o różnej długości"

#: ../duplicity/manifest.py:263
msgid "Manifests not equal because volume lists differ"
msgstr "Manifesty różne, bo zawierają różne listy woluminów"

#: ../duplicity/manifest.py:268
msgid "Manifests not equal because hosts or directories differ"
msgstr "Manifesty różne, bo dotyczący różnych adresów lub katalogów"

#: ../duplicity/manifest.py:425
msgid "Warning, found extra Volume identifier"
msgstr "Uwaga, wykryto dodatkowy identyfikator woluminu"

#: ../duplicity/manifest.py:451
msgid "Other is not VolumeInfo"
msgstr "Other nie jest typu VolumeInfo"

#: ../duplicity/manifest.py:454
msgid "Volume numbers don't match"
msgstr "Numery woluminów się nie zgadzają"

#: ../duplicity/manifest.py:457
msgid "start_indicies don't match"
msgstr "Wartości start_index są różne"

#: ../duplicity/manifest.py:460
msgid "end_index don't match"
msgstr "Wartości end_index są różne"

#: ../duplicity/manifest.py:467
msgid "Hashes don't match"
msgstr "Skróty są różne"

#: ../duplicity/patchdir.py:84 ../duplicity/patchdir.py:89
#, python-format
msgid "Patching %s"
msgstr "Aktualizowanie pliku %s"

#: ../duplicity/patchdir.py:550
#, python-format
msgid "Error '%s' patching %s"
msgstr "Błąd '%s' podczas aktualizacji pliku %s"

#: ../duplicity/patchdir.py:625
#, python-format
msgid "Writing %s of type %s"
msgstr "Zapisywanie %s typu %s"

#: ../duplicity/path.py:116
#, python-format
msgid "Warning: %s invalid devnums (0x%X), treating as (0, 0)."
msgstr ""
"Ostrzeżenie: plik specjalny %s ma niepoprawny nr urządzenia (0x%X); użycie "
"wartości (0, 0)."

#: ../duplicity/path.py:245 ../duplicity/path.py:306
#, python-format
msgid "Warning: %s has negative mtime, treating as 0."
msgstr "Ostrzeżenie: %s ma ujemny czas modyfikacji; użycie wartości 0."

#: ../duplicity/path.py:370
msgid "Difference found:"
msgstr "Wykryto różnicę:"

#: ../duplicity/path.py:379
#, python-format
msgid "New file %s"
msgstr "Nowy plik %s"

#: ../duplicity/path.py:382
#, python-format
msgid "File %s is missing"
msgstr "Plik %s nieodnaleziony"

#: ../duplicity/path.py:385
#, python-format
msgid "File %%s has type %s, expected %s"
msgstr "Plik %%s ma typ %s, ale oczekiwano %s"

#: ../duplicity/path.py:391 ../duplicity/path.py:417
#, python-format
msgid "File %%s has permissions %s, expected %s"
msgstr "Plik %%s ma uprawnienia %s, ale oczekiwano %s"

#: ../duplicity/path.py:396
#, python-format
msgid "File %%s has mtime %s, expected %s"
msgstr "Plik %%s ma czas modyfikacji %s, ale oczekiwano %s"

#: ../duplicity/path.py:404
#, python-format
msgid "Data for file %s is different"
msgstr "Zawartość pliku %s jest inna"

#: ../duplicity/path.py:412
#, python-format
msgid "Symlink %%s points to %s, expected %s"
msgstr "Dowiązanie symboliczne %%s odnosi się do %s, ale oczekiwano %s"

#: ../duplicity/path.py:421
#, python-format
msgid "Device file %%s has numbers %s, expected %s"
msgstr "Plik urządzenia %%s ma nr %s, ale oczekiwano %s"

#: ../duplicity/path.py:603
#, python-format
msgid "Making directory %s"
msgstr "Tworzenie katalogu %s"

#: ../duplicity/path.py:613
#, python-format
msgid "Deleting %s"
msgstr "Usuwanie %s"

#: ../duplicity/path.py:622
#, python-format
msgid "Touching %s"
msgstr "Odświeżanie czasu modyfikacji %s"

#: ../duplicity/path.py:629
#, python-format
msgid "Deleting tree %s"
msgstr "Usuwanie drzewa %s"

#: ../duplicity/robust.py:64
#, python-format
msgid "Error listing directory %s"
msgstr "Błąd uzyskania zawartości katalogu %s"

#: ../duplicity/selection.py:121
#, python-format
msgid "Skipping socket %s"
msgstr "Pomijanie pliku gniazda %s"

#: ../duplicity/selection.py:125
#, python-format
msgid "Error initializing file %s"
msgstr "Błąd inicjalizacji pliku %s"

#: ../duplicity/selection.py:129 ../duplicity/selection.py:154
#: ../duplicity/selection.py:454
#, python-format
msgid "Error accessing possibly locked file %s"
msgstr "Błąd dostępu do (prawdopodobnie) zastrzeżonego pliku %s"

#: ../duplicity/selection.py:169
#, python-format
msgid "Warning: base %s doesn't exist, continuing"
msgstr "Ostrzeżenie: ścieżka bazowa %s nie istnieje, kontynuowanie"

#: ../duplicity/selection.py:172 ../duplicity/selection.py:190
#: ../duplicity/selection.py:193
#, python-format
msgid "Selecting %s"
msgstr "Wybrano %s"

#: ../duplicity/selection.py:290
#, python-format
msgid ""
"Fatal Error: The file specification\n"
"    %s\n"
"cannot match any files in the base directory\n"
"    %s\n"
"Useful file specifications begin with the base directory or some\n"
"pattern (such as '**') which matches the base directory."
msgstr ""
"Krytyczny błąd: specyfikacja pliku\n"
"    %s\n"
"nie pasuje do żadnego pliku w bazowym katalogu\n"
"    %s\n"
"Użyteczne specyfikacje zaczynają się od ścieżki bazowej lub zawierają\n"
"wzorzec (np. \"**\"), który odpowiada ścieżce bazowej."

#: ../duplicity/selection.py:299
#, python-format
msgid ""
"Fatal Error while processing expression\n"
"%s"
msgstr ""
"Krytyczny błąd podczas przetwarzania wyrażenia\n"
"%s"

#: ../duplicity/selection.py:308
#, python-format
msgid ""
"Last selection expression:\n"
"    %s\n"
"only specifies that files be included.  Because the default is to\n"
"include all files, the expression is redundant.  Exiting because this\n"
"probably isn't what you meant."
msgstr ""
"Ostatnie wyrażenie:\n"
"    %s\n"
"sugeruje, że tylko te pliki zostaną wybrane. Ponieważ standardowo\n"
"wszystkie pliki są dodawane, wyrażenie to jest niepotrzebne. Kończę "
"wykonywanie,\n"
"ponieważ prawdopodobnie nie jest to to co chciałeś uzyskać."

#: ../duplicity/selection.py:364
#, python-format
msgid "Reading globbing filelist %s"
msgstr "Wczytywanie pliku z listą wyrażeń regularnych %s"

#: ../duplicity/selection.py:397
#, python-format
msgid "Error compiling regular expression %s"
msgstr "Błąd kompilacji wyrażenia regularnego %s"

#: ../duplicity/tempdir.py:160
#, python-format
msgid "Using temporary directory %s"
msgstr "Używanie tymczasowego katalogu %s"

#: ../duplicity/tempdir.py:204
#, python-format
msgid "Registering (mktemp) temporary file %s"
msgstr "Rejestrowanie (mktemp) pliku tymczasowego %s"

#: ../duplicity/tempdir.py:226
#, python-format
msgid "Registering (mkstemp) temporary file %s"
msgstr "Rejestrowanie (mkstemp) pliku tymczasowego %s"

#: ../duplicity/tempdir.py:258
#, python-format
msgid "Forgetting temporary file %s"
msgstr "Usuwanie informacji o pliku %s"

#: ../duplicity/tempdir.py:261
#, python-format
msgid "Attempt to forget unknown tempfile %s - this is probably a bug."
msgstr ""
"Próba usunięcia informacji o nieistniejącym pliku tymczasowym %s - jest to "
"pewnie błąd duplicity."

#: ../duplicity/tempdir.py:280
#, python-format
msgid "Removing still remembered temporary file %s"
msgstr "Usuwanie zapamiętanego pliku tymczasowego %s"

#: ../duplicity/tempdir.py:283
#, python-format
msgid "Cleanup of temporary file %s failed"
msgstr "Usuwanie tymczasowego pliku %s nie udało się"

#: ../duplicity/tempdir.py:288
#, python-format
msgid "Cleanup of temporary directory %s failed - this is probably a bug."
msgstr ""
"Usuwanie tymczasowego katalogu %s nie udało się - jest to pewnie błąd "
"duplicity."

#: ../duplicity/util.py:144
#, python-format
msgid "IGNORED_ERROR: Warning: ignoring error as requested: %s: %s"
msgstr ""
"IGNORED_ERROR: Ostrzeżenie: ignorowanie błędu zgodnie z intencją %s: %s"

#: ../duplicity/util.py:211
#, python-format
msgid "Releasing lockfile %s"
msgstr "Odblokowywanie pliku zamka %s"

#~ msgid ""
#~ "Warning: exclude-device-files is not the first selector.\n"
#~ "This may not be what you intended"
#~ msgstr ""
#~ "Ostrzeżenie: exclude-device-files nie jest pierwszym wzorcem.\n"
#~ "Niewykluczone, że nie jest to to, co chciałeś osiągnąć."
